dcos/dcos-ui
# chore(typescript): update version to match UI kit

## Testing

1. Check this branch out (`git checkout danielmschmidt/DCOS-39297`)
2. Rebase a branch that uses the Cell component on top of it. (e.g. `git rebase danielmschmidt/DCOS-39069 `)
3. Activate the new NodesTable

Change this line in `NodesTableContainer.js`

```diff
-import NodesTable from "../../../components/NodesTable";
+import NodesTable from "../../../components/NodesTable-new";
```

## Trade-offs



- There might be other ways to solve this issue that I don't know of, but all my research hinted in this direction


## Dependencies



None
